processor/src/main/java/org/mapstruct/ap/internal/model/source/SourceMethod.java

```diff
@@ -590,6 +592,11 @@ public boolean isBeforeMappingMethod() {
         return Executables.isBeforeMappingMethod( getExecutable() );
     }
 
+    public boolean isAbstract() {
```

Comment:
although `isAbstract` might then not be a good name.. I'll change that to `isAbstractOrDefault`.